src/test/kotlin/com/expedia/graphql/generator/extensions/KClassExtensionsTest.kt

```diff
@@ -102,6 +126,12 @@ open class KClassExtensionsTest {
         assertEquals(listOf("publicFunction"), properties.map { it.name })
     }
 
+    @Test
+    fun `test getting functions from absctract classes`() {
```

Comment:
typo
```suggestion
    fun `test getting functions from abstract classes`() {
```

Reply:
Nice, the suggestion tool is helpful

Reply:
ohh you can directly commit from it. even better